applications/GettingStartedWith/DateRangeBox/index.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+    $("#dateRangeBox").dxDateRangeBox({
+        min: new Date(1920, 0, 1),
+        max: new Date(2030, 0, 1),
+        startDateLabel: "Start",
+        endDateLabel: "End",
+        labelMode: "floating",
+        displayFormat: "EEEE, d 'of' MMM", // "Tuesday, 19 of Oct - Sunday, 24 of Oct" 
+        useMaskBehavior: true,
```

Comment:
I suppose it's a good idea to set `openOnFieldClick: false` when using masks behavior. Otherwise when you click on some part of input Popup will open, and you won't be able to control value in inputs wia keyborad arrows (you will navigate in calendar instead)